routes/root.js

```diff
@@ -91,8 +91,8 @@ route.get('/stats', (req, res) => {
       res.render('pages/stats', {
         participants: data[0],
```

Comment:
lets's do it for all the values

Reply:
Yes, moved the logic to getCounts functions as it's better to have the check there only. Ref #182